code/modules/research/designs/autolathe_designs.dm

```diff
@@ -856,12 +856,20 @@
 	build_path = /obj/item/ammo_casing/shotgun/rubbershot
 	category = list("hacked", "Security")
 
-/datum/design/shotgun_laser
```

Comment:
is the laser shell design being removed on purpose?

Reply:
I actually placed it there a few months back during laser shells and all IIRC. Now that i'm doing a quick fix for sec arsenal (and tiders if they get their claws on a shotty) i figured i might as well keep the laser shells which replaced buckshot for the while restricted to the seclathe.